src/status_im/contexts/centralized_metrics/tracking.cljs

```diff
@@ -3,37 +3,61 @@
     [legacy.status-im.utils.build :as build]
     [react-native.platform :as platform]))
 
-(defn user-journey-event
-  [action]
+(defn key-value-event
+  [event-name val-key value]
   {:metric
-   {:eventName  "user-journey"
+   {:eventName  event-name
     :platform   platform/os
     :appVersion build/app-short-version
-    :eventValue {:action action}}})
+    :eventValue {val-key value}}})
+
+(defn user-journey-event
+  [action]
+  (key-value-event "user-journey" :action action))
+
+(defn navigation-event
+  [view-id]
+  (key-value-event "navigation" :viewId view-id))
 
 (def ^:const app-started-event "app-started")
-(def ^:const navigate-to-create-profile-event "navigate-to-create-profile")
-(def ^:const communities-tab-clicked "communities-tab-clicked")
-(def ^:const wallet-tab-clicked "wallet-tab-clicked")
-(def ^:const chats-tab-clicked "chats-tab-clicked")
+
+(def ^:const view-ids-to-track
+  #{;; Tabs
+    :communities-stack
+    :chats-stack
+    :wallet-stack
+
+    ;; Onboarding
+    :screen/onboarding.intro
+    :screen/onboarding.new-to-status
+    :screen/onboarding.sync-or-recover-profile
+    :screen/onboarding.enter-seed-phrase
+    :screen/onboarding.create-profile
+    :screen/onboarding.create-profile-password
+    :screen/onboarding.enable-biometrics
+    :screen/onboarding.generating-keys
+    :screen/onboarding.enable-notifications
+    :screen/onboarding.sign-in-intro
+    :screen/onboarding.sign-in
+    :screen/onboarding.syncing-progress
+    :screen/onboarding.syncing-progress-intro
+    :screen/onboarding.syncing-results
+    :screen/onboarding.welcome})
 
 (defn track-view-id-event
   [view-id]
-  (case view-id
-    :communities-stack (user-journey-event communities-tab-clicked)
-    :chats-stack       (user-journey-event chats-tab-clicked)
-    :wallet-stack      (user-journey-event wallet-tab-clicked)
-    nil))
+  (when (contains? view-ids-to-track view-id)
+    (navigation-event (name view-id))))
 
 (defn tracked-event
   [[event-name second-parameter]]
   (case event-name
-    :onboarding/navigate-to-create-profile
-    (user-journey-event navigate-to-create-profile-event)
-
```

Comment:
Why are we removing this track event?

Reply:
Because it's being tracked a navigation event, so we don't need to track the click.